app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -108,17 +114,36 @@ class IssueTable
         data: data,
         dataType: 'json',
         success: (data) ->
+          $that.prop('checked', false)
           issue_id = $that.val()
 
           $($row.find('td')[that.tagColumnIndex]).replaceWith(data.tag_cell)
           $("#issues #issue_#{issue_id}").replaceWith(data.issue_link)
-          if $('input[type=checkbox]:checked').length == 0
-            $('.js-issue-actions').css('display', 'none')
+          if $('input[type=checkbox]:checked.js-multicheck').length == 0
+            that.resetToolbar()
 
         error: (foo,bar,foobar) ->
           $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class='text-error'>Please try again</td>")
       }
 
+  onMergeSelected: (event) ->
```

Comment:
If my understanding is correct, in this part, you're creating a form with the issue ids and then you submit the form to the "merging" step. I think it's unnecessary for the merging action to be `POST` since there is no change in state yet. Instead, you can use `GET` and construct the merging url as such: `issues/merging?issue_id[]=1&issue_id[]=2&issue_id[]=3`